command/agent/command.go

```diff
@@ -618,6 +618,15 @@ func (c *Command) Run(args []string) int {
 		return 1
 	}
 
+	// GH-9489: Reset UI to prevent prefixed json output
```

Comment:
We don't include GitHub issue IDs in comments. Source control provides that context.

```suggestion
	// reset UI to prevent prefixed json output
```